.mocharc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "spec": "packages/*/!(integration-tests)/test/*.js",
+  "spec": "packages/*/!(integration-tests)/test/{*.js,**/*.{test,spec}.js}",
   "require": ["@parcel/babel-register", "@parcel/test-utils/src/mochaSetup.js"],
   // TODO: Remove this when https://github.com/nodejs/node/pull/28788 is resolved
```

Comment:
Unrelated to your change, but I was curious and it looks like this issue was closed in 2019 :P

Reply:
I'm not going to fix this one here

Reply:
Actually let's resolve it